### PR TITLE
#157507751 Setup e2e

### PR DESCRIPTION
**Pull request by kevinejiro**
#### What does this PR do?
Setup End to End testing. 

#### Description of Task to be completed?
- add .babelrc file to configure presets and plugins
- install nightwatch for e2e test
- add nightwatch configuration files
- add index.html file
- install http-server to load .html file in browser
- install chromedriver to control chrome
- add test.js for e2e test
- add scripts start http-server and nightwatch

#### How should this be manually tested?
This can be manually tested by running the `start:mock-server` script to start the server and  `test:e2e` script on another terminal to run e2e test.

#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?
#[157507751](https://www.pivotaltracker.com/n/projects/2170023/stories/157507751)

### CODE REVIEW

**Comment by andela-kakpobome** [comment]
@kevinejiro use sentence case for your PR titles i.e `#157507751 Setup e2e` and also resolve the conflicts

**Comment by andela-kakpobome** [comment]
Lastly, let your story ID be a link to the story on Pivotal tracker

File: src/index.html [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
```
[the remark this answers is not in the document]
Comment (kevinejiro) [reply src/index.html]: Thanks for the feedback, ill fix this promptly

File: src/index.html [hunk 2]
```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+
```
[the remark this answers is not in the document]
Comment (kevinejiro) [reply src/index.html]: thanks for the feedback, i have implemented this

File: src/index.html [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    <title>
+      Ativo
+    </title>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+  </head>
+
+  <body>
+    <div>
+      <h1 class="Header">
```
Comment (andela-kakpobome) [inline src/index.html]: Why is your class name capitalized?

**Comment by andela-kakpobome** [comment]
@kevinejiro how are you still sending commits that don't follow the conventions? What more communication do you require before you conform to them?

**Comment by missating** [comment]
@kevinejiro Please resolve conflicts

**Pull request merged**